crates/forge/src/init.rs

```diff
@@ -98,13 +98,23 @@ pub fn run(project_name: &str) -> Result<()> {
             .current_dir(current_dir)
             .arg("new")
             .arg(&project_path)
-            .env("SCARB_INIT_TEST_RUNNER", "starknet-foundry")
+            .env("SCARB_INIT_TEST_RUNNER", "cairo-test")
```

Comment:
Why

Reply:
So scarb doesn't call `snforge init` again under the hood https://github.com/software-mansion/scarb/blob/09590f5fc07365c3e4188b0e8fa2e774a8269d68/scarb/src/ops/new.rs#L220. We are replacing the template generated anyway.